### PR TITLE
Correction of bug KeyError: 'ID' in OneFichierCom.py.

### PR DESCRIPTION
**Pull request by thadrien**

### CODE REVIEW

File: module/plugins/hoster/OneFichierCom.py [hunk 1]
```diff
@@ -99,8 +99,8 @@ def handle_direct(self, pyfile):
     def handle_free(self, pyfile):
         self.check_errors()
 
-        id = self.info['pattern']['ID1'] or self.info['pattern']['ID2']
-        url, inputs = self.parse_html_form('action="https://1fichier.com/\?%s' % id)
+        id = self.info['pattern']['N']  # Check me : should this one be the file name or the onefichier name ???
+        url, inputs = self.parse_html_form('action="https://1fichier.com/\?[a-zA-Z0-9]+')
```
Comment (thadrien) [inline module/plugins/hoster/OneFichierCom.py]: This lines doesn't download a file.

**Comment by thadrien** [comment]
This line doesn't download the page source, already downloaded, but search the "file link" into the page source. Regex is precisely the right tool for this.
Have a nice day.

**Comment by joberreiter** [comment]
Ah, my mistake, i see.

**Comment by thadrien** [comment]
You're welcome.
However, I'm not sure I picked the right id, even if the files downloads pretty nice.

**Comment by vuolter** [comment]
THX.

**Comment by GammaC0de** [comment]
`id` is totally unneeded anymore so I issued #1995

**Pull request merged**